### PR TITLE
feat(lts/transfer): new data source to query log transfers

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
Supports a new data source that used to query the log transfers.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. supports a new data source that used to query the log transfers.
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/lts' TESTARGS='-run=TestAccDatasourceTransfers_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/lts -v -run=TestAccDatasourceTransfers_basic -timeout 360m -parallel 4
=== RUN   TestAccDatasourceTransfers_basic
=== PAUSE TestAccDatasourceTransfers_basic
=== CONT  TestAccDatasourceTransfers_basic
--- PASS: TestAccDatasourceTransfers_basic (76.85s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/lts       76.919s
```

### CODE REVIEW

File: docs/data-sources/lts_transfers.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+subcategory: "Log Tank Service (LTS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_lts_transfers"
+description: ""
+---
```
Comment (jinyangyang222) [inline docs/data-sources/lts_transfers.md:6]:
```suggestion
description: |-
  Using this data source to query the list of log transfers within HuaweiCloud.
---
```

File: huaweicloud/services/lts/data_source_huaweicloud_lts_transfers.go [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+
+	getTransferOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		MoreHeaders:      map[string]string{"Content-Type": "application/json"},
+	}
+
+	// The query parameter 'offset' is unavailable
+	requestResp, err := client.Request("GET", listPath, &getTransferOpt)
+	if err != nil {
+		return diag.Errorf("error querying log transfer: %s", err)
```
Comment (jinyangyang222) [inline huaweicloud/services/lts/data_source_huaweicloud_lts_transfers.go:292]:
```suggestion
		return diag.Errorf("error querying log transfers: %s", err)
```

Comment (Lance52259) [reply huaweicloud/services/lts/data_source_huaweicloud_lts_transfers.go]: done

File: docs/data-sources/lts_transfers.md (hunk 1 above)
Comment (Lance52259) [reply docs/data-sources/lts_transfers.md]: done.

**Pull request merged**